config.json

```diff
@@ -18,6 +18,17 @@
 	    ]
 	},
 	"propagatedmount": "/mnt",
+	"mounts": [
+		{
+		"name": "KeyPath",
+		"description": "Path to SSH keys",
+		"source": "/tmp",
+		"destination": "/tmp/sshfs",
```

Comment:
@cyli I am just wondering if we can provide the credentials to the instance of the volume, rather than to the whole plugin. Whether we make that available to the plugin container, to the user container or elsewhere is an implementation detail.

I guess the main issue is that we pluck `--identity` from the plugin mount, rather than a location associated with the particular volume mount.

I think we are missing a something here to make this work correctly.